### PR TITLE
added LOG function

### PR DESCRIPTION
**Pull request by jatingarg**
Hi yathura, i am relatively new to github but i know java programming and would like to contribute to your project. Do tell me if its correctly done. :)

### CODE REVIEW

**Comment by yuthura** [comment]
Hi @jatingarg, I appreciate your effort to contribute and can congratulate you on a job wel done forking my repository and creating this pull request. Unfortunately this repository is still in (relatively) early development and I'm still actively changing the inner workings of the library (which makes it hard for other people to contribute to this repository). I have also not yet added a "how to contribute" section to the readme page, which might come soon (which will explain my wishes and expectations for contributors).

As you can see with issue #20 and pull request #30, I recently decided (and just today actually proceeded) to _remove_ the function-specific classes in favor of generic base classes used by the helper methods in `Helpers`. So I'm sorry to say I will decline this pull request, but if you truly want to contribute, feel free to implement the `log` function to the `Helper` class (see issue #22 for more mathematical functions I still need to add).

I would like to ask you to create a branch other than development to perform your changes on (i.e. `git checkout -b feature/22-description-of-your-changes`) which would fit nicely into my workflow based on http://nvie.com/posts/a-successful-git-branching-model/ (and which will most likely be explained in the readme/wiki in the future).

Unfortunately I have only limited time to work on this personal project (and my other projects here on Github), so I can't say upfront when I will be working on it.

File: src/com/github/yuthura/bianca/functions/log.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.github.yuthura.bianca.functions;
+
+public class log extends UnaryFunction {
```
Comment (yuthura) [inline src/com/github/yuthura/bianca/functions/log.java]: Please follow publicly accepted Java coding conventions and begin all class names with a capital (i.e. `Log`) instead.

**Comment by jatingarg** [comment]
@yuthura, Thanks for all the suggestions, I will keep the conventions in my mind, Also as you said i will create a separate branch other than development for my changes and study the branching model. Also I will look into the Helper Class and see what can be done with the mathematical functions and come back to you with an update.And feel free to respond to it whenever u want, we all have time constraints.Keep up the good work!

**Comment by yuthura** [comment]
Thanks for understanding @jatingarg, good luck and I'll be sure to check out any updates! :smiley:

**Pull request closed without merging**